src/Application/AnalyzeCommand.php

```diff
@@ -383,7 +388,15 @@ function ($renderPoints) {
             );
         }
 
-        return $result;
+        if ($generateBaselineFile) {
+            $this->generateBaseline($result, $output, $generateBaselineFile);
+        }
+
+        $ignoredErrorHelperResult = $this->ignoredErrorHelper->initialize();
+
+        $ignoredErrorHelperProcessedResult = $ignoredErrorHelperResult->process($result->errors, true, $twigFileNames, false);
```

Comment:
I think it would make more sense to merge these 2 things into a single IgnoredErrorProcessor class.